src/applications/pensions/tests/unit/chapters/05-financial-information/netWorthEstimation.unit.spec.jsx

```diff
@@ -52,7 +51,7 @@ describe('Financial information net worth estimation pension page', () => {
     pageTitle,
   );
 
-  it('should show warning', async () => {
+  it.skip('should show warning', async () => {
```

Comment:
skipped test to be addressed in [future issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/114320)